src/PlaneWaveBasis.jl

```diff
@@ -106,10 +106,13 @@ struct PlaneWaveBasis{T,
     ## Symmetry operations that leave the discretized model (k and r grids) invariant.
     # Subset of model.symmetries.
     symmetries::Vector{SymOp{VT}}
-    # Whether the symmetry operations leave the rgrid invariant
+    # Whether the symmetry operations leave the rgrid invariant.
     # If this is true, the symmetries are a property of the complete discretized model.
     # Therefore, all quantities should be symmetric to machine precision
     symmetries_respect_rgrid::Bool
+    # Whether the symmetry operations leave the basis invariant. Otherwise the symmetry
+    # operations will leave the lattice invariant.
+    symmetries_respect_basis::Bool
```

Comment:
This is just a very confusing name. Should rather be respect_BZ or respect_kgrid, no ?

Also the boolean flags maybe go out of hand here. Maybe it's better to have a separate struct that collect the symmetry properties (preserves lattice, preserves kgrid, preserves structure, etc.). That might also make it easier for users to specify cases where exactly one symmetry is broken (e.g. along a displacement vector). Just a thought. Feel free to discard it if this is too much work.